project_timesheet_holidays_contract/readme/DESCRIPTION.md

```diff
@@ -0,0 +1 @@
+Module allows to take into account employee contract along with provided end date when timesheet on time off.
```

Comment:
Can you elaborate here a bit?

Reply:
@yankinmax

Reply:
Hello, @rousseldenis I don't have time now, sorry.